main.py

```diff
@@ -28,6 +32,21 @@ def __repr__(self):
     def tovoice(self):
         return f'{self.name}, {self.number},'
 
+class State(Base):
+    __tablename__ = 'STATE'
+
+    id = Column(Integer, primary_key=True)
+    state = Column(String)
```

Comment:
try representing state as:

id
key
value

where key is text and value is also text

Reply:
then consider writing a function getState() which returns the state settings, in a dictionary.

and also consider writing a setState function which inserts (or updates) the state on the database.
Also getStateByName(key) should be useful at some point I guess..

Reply:
these functions exist as get_state and set_stat, I've renamed them.